diffpy/snmf/new_subroutines.py

```diff
@@ -5,6 +5,17 @@
 from diffpy.snmf.componentsignal import ComponentSignal
```

Comment:
the module is better if it, in principle, can contain more containers, so I would call it `containers.py` then it can have `ComponentSignal` but also any others that we might come up with later.